example/cr-detailed.yaml

```diff
@@ -12,6 +12,13 @@ spec:
       pullPolicy: IfNotPresent
 
     replicas: 3
+    resources:
```

Comment:
Is there any test that validates that these values are properly passed to the process?

Reply:
No, we will work on adding a test to validate that resources are passed down to pod templates correctly.

Reply:
Test added. Could you please review it again? Thanks!